tests/test_commands.py

```diff
@@ -874,6 +874,16 @@ def test_pttl_no_key(self, r):
         "PTTL on servers 2.8 and after return -2 when the key doesn't exist"
         assert r.pttl('a') == -2
 
+    @skip_if_server_version_lt('6.2.0')
+    def test_hrandfield(self, r):
+        assert r.hrandfield('key') is None
+        r.hset('key', mapping={'a': 1, 'b': 2, 'c': 3, 'd': 4, 'e': 5})
+        assert r.hrandfield('key') is not None
```

Comment:
Nit: Can the comments for each test happen before the test with a newline? eg:

assert len(r.hrandfield('key', 2)) == 2

# with values
assert len(r.hrandfield('key', 2, True)) == 4

# without duplications
assert len(r.hrandfield('key', 10)) == 5